connection.php

```diff
@@ -0,0 +1,725 @@
+<?php
+
+/**
+* @author: mfsi-krishnadev
+* @class : database connection and operation
+* 
+*/
+class db_connection 
+{
+	//declaration of variables
+	private $hostname = "localhost";
+	private $username = "root";
+	private $password = "mindfire";
+	private $database = "employee";
+	public  $connect;
+
+	//contructor to initialse the variable
+	function __construct()
+	{
+		$this->connect = mysqli_connect($this->hostname,$this->username,$this->password,$this->database);
+
+		//Error Handling
+		if(mysqli_connect_error())
+		{
+			trigger_error("Failed to connect mySQL:". mysqli_connect_error(),E_USER_ERROR);
```

Comment:
space should be present before and after a operator.
here space is present after dot but not present before space.